src/components/Section/ProposalVoting/ProposalVotingSection.tsx

```diff
@@ -68,56 +82,71 @@ const ProposalVotingSection = ({
         delegators,
         account,
         ownVotingPower,
-        delegatedVotingPower,
+        delegatedVp,
         voteDifference
       ),
-    [testData]
+    // TODO remove test data
+    [testData ? testData : [votes, choices, delegate, delegators, account, ownVotingPower, delegatedVp, voteDifference]]
+  )
+  const { votesByChoices, totalVotes } = useMemo(
+    () => getPartyVotes(delegators, votes, choices),
+    [delegators, votes, choices]
   )
-  const { votesByChoices, totalVotes } = useMemo(() => getPartyVotes(delegators, votes, choices), [testData])
 
+  loading = loading || accountState.loading || delegationState.loading || vpOnProposalState.loading
```

Comment:
Please don't reassign props 🙏  Maybe loading from props should have a different name. What is it loading? The proposal?

Reply:
loading is `voting || proposalState.loading || votesState.loading || isLoadingVotingPower`.

Maybe `const proposalVotingSectionLoading = loading || accountState.loading || delegationState.loading || vpOnProposalState.loading`